### DISCUSSION TITLE
Similar Add-in for Excel

### DISCUSSION THREAD

**Discussion by sharky98** [joyfullservice/msaccess-vcs-addin#524] [General]
Hello,

I was wondering if anyone know of a similar add-in for Excel?

If not, do you think that this MS Access Add-in could be adapted for Excel (like dropping the part regarding the tables, SQL, etc.)?

Thanks!

**Comment by hecon5** [comment]
We use https://rubberduckvba.com/ for import/export outside of Access. It works great for things which aren't Access. It can get a bit bogged down in Access sometimes in large projects (especially in Access), but for the most part, it works just great for me. 

![image](https://github.com/user-attachments/assets/7082e0ef-8ca7-4a48-90f2-6154c53d2602)
